packages/core/review-workflows/admin/src/routes/content-manager/model/id/components/StageSelect.tsx

```diff
@@ -160,13 +164,26 @@ export const StageSelect = () => {
         name={STAGE_ATTRIBUTE_NAME}
         id={STAGE_ATTRIBUTE_NAME}
       >
-        <Field.Label>
-          {formatMessage({
-            id: 'content-manager.reviewWorkflows.stage.label',
-            defaultMessage: 'Review stage',
-          })}
-        </Field.Label>
+        {isCompact ? (
+          <VisuallyHidden>
+            <Field.Label>
+              {formatMessage({
+                id: 'content-manager.reviewWorkflows.stage.label',
+                defaultMessage: 'Review stage',
+              })}
+            </Field.Label>
```

Comment:
Great a11y stuff! I'd just recommend storing the formattedMessage in a const above to avoid writing it twice